packages/strapi-plugin-users-permissions/admin/src/components/Policies/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { useMemo } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { get, isEmpty, takeRight, toLower, without } from 'lodash';
+import { InputsIndex as Input } from 'strapi-helper-plugin';
+
+import { useUsersPermissions } from '../../contexts/UsersPermissionsContext';
+import BoundRoute from '../BoundRoute';
+import { Header, Wrapper } from './Components';
+
+const Policies = () => {
+  const { selectedAction, routes, permissions, policies, onSelectedPolicy } = useUsersPermissions();
+  const baseTitle = 'users-permissions.Policies.header';
+  const title = !selectedAction ? 'hint' : 'title';
+  const path = without(selectedAction.split('.'), 'controllers');
```

Comment:
If you don't need the `controllers` why are you setting it in the `handleSelectPolicy` method in the `Permissions.PermissionRow.SubCategory.index` component.